### ISSUE TITLE
loss = sum(layer_losses)

### ISSUE THREAD

**Issue by 0x54daydreamer** [leongatys/PytorchNeuralStyleTransfer#16]
Hello, I am new to pytorch and this scene in general, but when I run the code I get an error;torch.sum()  Use tensor.detach().numpy() instead.

I am not familiar with how to detach or couldnt find an example of detaching each tensor on the list.
Is the following equivalent code?
:
for elem in layer_losses[1:]:
            loss += elem

**Comment by jianlgler** [comment]
Same problem here

**Comment by ankit7699** [comment]
loss = torch.sum(torch.stack(layer_losses))
